readme.md

```diff
@@ -16,6 +16,28 @@ We have tutorials on our [Developer Site](https://dev.mixer.com) to help you get
 - [Chat Tutorial](https://dev.mixer.com/tutorials/chatbot.html)
 - [REST API Tutorial](https://dev.mixer.com/tutorials/rest.html)
 
+### Visual Studio Code Reference Launch Config
```

Comment:
Can you put this under a `Contributing` section or similar? To make it clear to consumers that this is for if they want to iterate on the client itself, and isn't related to any particular debugging setup they need if they're consuming this.